### DISCUSSION TITLE
co dodac lub usuniac w filtrach.

### DISCUSSION THREAD

**Discussion by ghost** [MajkiIT/polish-ads-filter#19550] [General]
co dodac lub usuniac?

szczegolne mi chodzo o to: "Jeżeli korzystasz z Polskich Filtrów Elementów Irytujących, to nie dodawaj listy RODO-Ciasteczkowej, ponieważ Filtry RODO-Ciasteczkowe znajdują się już w Filtrach Elementów Irytujących."

![filtry](https://user-images.githubusercontent.com/17565773/114223965-bf4fd880-9970-11eb-8718-6c3d3255a920.jpg)

**Comment by hawkeye116477** [comment]
Usuwać napewno nic. Ale najważniejszej listy brak, gdzie nasz polski KAD? 

Polecam przejrzeć https://majkiit.github.io/polish-ads-filter/, możesz przefiltrować wg kategorii.

**Comment by ghost** [comment]
co to kad?

**Reply by hawkeye116477** [reply]
No co ty, nie znasz jednej z najważniejszych polskich list, żartujesz?

W takim razie mam dla ciebie lekturę na dziś => https://kadantiscam.netlify.app/.

**Comment by ghost** [comment]
aha, nie wiedzialem, ze to jest wazny filtr.
